docs/development.md

```diff
@@ -92,27 +92,50 @@ list so that they will see CTRs as they come through. If another
 committer responds with a -1, the commit should be rolled back and the
 formal RTC process should be followed.
 
+### Enable Automatic Backporting
+
+Before merging the pull request, it should be decided whether the contribution should be backported to other release
+branches.
+This should be the case for most non-breaking changes.
+If the change applies to something that is not present at all on other still supported release branches, like updating
+a dependency that was only introduced on `master` or code that was added / heavily modified on `master`, then
+backporting cannot work or would require too much effort.
+Otherwise, the appropriate labels for backporting should be added before the pull requests gets merged as that allows
```

Comment:
What milestone shall we use? Shall we use the backported branch's milestone (right now it's 0.6 milestone)?

Reply:
When I wrote this, two labels where necessary: `backport` and `backport/v0.6`. After #3297 is merged, only `backport/v0.6` needs to be added. I'll amend this to include that as an example and mention that the label(s) depend on which branches we currently support.

Reply:
Oh sorry I was not talking about labels. Maybe I should have started the conversation somewhere else, but I meant to ask about **milestones**.

Reply:
In other words, for a new PR that has backport label, shall we mark it as 1.0 milestone or 0.6 milestone?

Reply:
The PR should get the milestone for its target branch. So, a PR on `master` should get the 1.0.0 milestone and a PR for `v0.6` should get the 0.6.3 milestone. This also applies to backporting PRs.
That way, each milestone will really list all PRs that are included in that release and we don't need this "This release also includes all changes from the 0.6.z release" any more.

I'm however not sure about issues that are closed by a PR that will be backported. We can only add an issue to one milestone. Should that be 1.0.0 or 0.6.3?